public/js/reducers/purchase.js

```diff
@@ -1,9 +1,9 @@
 import * as actionTypes from 'constants/action-types';
 
 
-export const initialState = {
+export const initialPurchaseState = {
   completed: false,
-  payment_methods: [],
+  payMethods: null,
```

Comment:
what's the point of null as opposed to an empty list? Just for consistency? It looks like it makes Purchase a little more tricky since the code has to defend against nulls. The reason null was used to initialize subscriptions was just to indicate that none had been loaded yet. For the purchase flow that wouldn't really apply (they are loaded at sign-in).